CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 - Updated: Added additional resources to the operators group policy for MFA
 - Fix: Adjusted values displayed in the CloudWatch dashboard, so take into consideration doubled logs in ApiGateway logs 
+- Added: Security Groups for AWS Quick Sight support 
```

Comment:
We add these at the top of the list, so is reverse chronoligical order

Reply:
fixed